tests/unittests/Foundation/NSOrderedSetTests.mm

```diff
@@ -0,0 +1,324 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <Foundation/Foundation.h>
+#import <TestFramework.h>
+
+void assertOrderedSetContent(NSOrderedSet* set, NSObject* first, ...) {
```

Comment:
this function needs to be static or better yet shared between files. As is it breaks the one definition rule as C style functions are extern by default meaning both Translations units will see the definitions and the linker will just choose one for you. That is "fine" as long as they stay the exact same but that begs the question why they just aren't sharing the implementation to begin with.